clusterloader2/testing/density/config.yaml

```diff
@@ -90,7 +96,27 @@ steps:
   - Identifier: WaitForLatencyPodRunning
     Method: WaitForRunningPods
     Params:
-      desiredPodCount: {{.Nodes}}
+      desiredPodCount: {{$totalLatencyPodsRounded}}
+      labelSelector: kind = latencyPod
+      timeout: 10m
+# Remove latency pods
+- phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: 0
+    tuningSet: Uniform5qps
+    objectBundle:
+    - basename: latency-pod-rc
+      objectTemplatePath: rc.yaml
```

Comment:
If replicasPerNamespace is 0 (or in general smaller than before so that it results in deletion of objects),
do we really need to pass objectTemplatePath and templateFillMap?
Conceptually this shouldn't be needed.

Reply:
> do we really need to pass objectTemplatePath (...)?

Yes. Previously there was `ObjectKind` structure that described kind and api group. Currently this data has to be taken from yaml file. It required due to possible names collision.

> do we really need to pass (...) templateFillMap?

No. It's not required. I've removed it from test.

Reply:
OK thanks. That makes sense.